.github/workflows/main.yml

```diff
@@ -1,20 +1,19 @@
 name: Build App
 
-on: 
+on:
   push:
-    branches: 
+    branches:
       - master
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - name: Run a one-line script
-      uses: peter-evans/repository-dispatch@v1
-      with:
-        token: ${{ secrets.ACCESS_TOKEN }}
-        repository: paritytech/substrate.io
-        event-type: content-update
-        client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
+      - name: Run a one-line script
```

Comment:
```suggestion
      - name: Run Prettier
```

Reply:
See my comment above; we shouldn't be changing anything like this as part of this PR because nothing is changing except formatting.